client/src/components/LoggedOut/LoggedOut.jsx

```diff
@@ -1,42 +1,40 @@
 import * as React from "react"
 import "./LoggedOut.css"
-import axios from "axios"
-import { useState, useEffect } from "react"
 import LoginForm from "../LoginForm/LoginForm"
-import header from './test.png'
-import aboutImg from './about.png'
+import { Parallax } from 'react-parallax'
+import bgImage from './coffee.jpg'
 
 export default function LoggedOut(props) {
     return (
         <div className="logged-out">
-            <div className="header">
-                <img src={header}/>
-            </div>
-            <div className={props.sessionToken ? "hidden" : "box1"}>
-                <div className="box">
-                    <LoginForm setSessionToken={props.setSessionToken} sessionToken={props.sessionToken}/>
+            <Parallax bgImage={bgImage} strength={-500} >
```

Comment:
great use of parallax to meet the visual design component and add polish to the UI.